### PR TITLE
Better project readme according to the GFM

### PR DESCRIPTION
**Pull request by resurtm**
GFM is http://github.github.com/github-flavored-markdown/

PHP highlighting now works. Preview is in my fork's branch: https://github.com/resurtm/shopping-cart-component/tree/better-readme

### CODE REVIEW

File: readme_ru.txt [hunk 1]
```diff
@@ -10,8 +10,8 @@ Shopping Cart
 ---------------------
 ### 1 вариант: Подключение через конфиг
 В `protected/config/main.php` добавить:
-~~~
-[php]
+```php
```
Comment (cebe) [inline readme_ru.txt:13]: think there has to be a newline before every ```
can you verify that this is the case?

**Comment by asifrafeeq** [comment]
it will b highly appreciable if you provide the database tables also.... thanks a lot.

**Comment by resurtm** [comment]
@asifrafeeq, what do you mean? Do you want us to add DDL of sample table which is suitable for storing products?

**Comment by asifrafeeq** [comment]
yes of-course,,,

**Comment by cebe** [comment]
@resurtm any reason against merging this? why did you close it?

**Comment by resurtm** [comment]
@cebe, better to fix it directly. This is very minor non-critical issue. :-)

**Comment by asifrafeeq** [comment]
:) okey, sir I want to ask another question that how can I use the API which u have mentioned in this component, should I use this API code in controller or in view, plz guide me.

thanks a lot.

 
Asif Ullah 
contact # +923369742766
email id: asifrafeeq1@yahoo.com

**Pull request closed without merging**